.cirrus.yml

```
@@ -1,3 +1,17 @@
+FreeBSD_task:
+  freebsd_instance:
+    image_family: freebsd-12-1
+
+  install_script:
+    - pkg update
+    - pkg install -y cmake gmake openal-soft sdl2
+
+  script:
+    - mkdir build
+    - cd build
+    - cmake .. -DAQUARIA_USE_SDL2=ON
```

```suggestion
    - cmake .. -DAQUARIA_INTERNAL_PNG=OFF -DAQUARIA_USE_SDL2=ON
```

---

Alternatively, add explicit workaround
```suggestion
    - export CXXFLAGS=-isystem/usr/local/include
    - cmake .. -DAQUARIA_USE_SDL2=ON
```